boot/boot.js

```diff
@@ -1444,6 +1444,7 @@ $tw.loadTiddlersBrowser = function() {
 	for(var t=0; t<containerIds.length; t++) {
 		$tw.wiki.addTiddlers($tw.wiki.deserializeTiddlers("(DOM)",document.getElementById(containerIds[t])));
 	}
+	callback();
```

Comment:
For example, what's the difference between:

```js
$tw.loadTiddlersBrowser();
console.log('done');
```
and

```js
$tw.loadTiddlersBrowser(()=> console.log('done'))
```

Absolutely nothing.

The same happens with `$tw.loadTiddlersNode`. If you want this to make any sense you should make `$tw.loadTiddlersFromPath` load the files asynchronously, providing a list of tiddlers to the callback which will add the tiddlers to tw and then call the callback. That way, it will make sense